demo/test.c

```diff
@@ -1543,262 +1705,277 @@ static int test_mp_ilogb(void)
    int size;
    const uint32_t max_base = MP_MIN(UINT32_MAX, MP_DIGIT_MAX);
```

Comment:
The conversion check complaint here (https://travis-ci.org/libtom/libtommath/jobs/581756157)
(No, I don't know how the extra forward slash came into existence at that place)
My error?